components/clickup/sources/new-folder/new-folder.mjs

```diff
@@ -0,0 +1,31 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "clickup-new-folder",
+  name: "New Folder (Instant)",
+  description: "Emit new event when a new folder is created",
+  version: "0.0.1",
+  dedupe: "unique",
+  type: "source",
+  methods: {
+    ...common.methods,
+    _getMeta({ folder_id: folderId }) {
+      return {
+        id: folderId,
+        summary: folderId,
```

Comment:
Giving a summary more than `folderId` might be better for user experience.

Reply:
The webhook payload doesn't provide more information :(